rmarkdown/research/recommendation-of-adoption.md

```diff
@@ -0,0 +1,32 @@
+# Recommendation of Adoption
+
+We recommend RMarkdown for authoring literate research reports when the following conditions pertain:
+
+1. The report is for a client
+2. When the report contains graphs or statistics.
+3. When we use code to generate the graphs or statistics. If we are doing an quick analysis in Excel, because that is what a client needs, then a literate research report would not be useful approach.
+
+RMarkdown should be used even if it the report seems like it will be quick and lightweight. Experience tells us that it is not easy to predict when an analysis will grow in complexity or when a client may return months later to ask about a detail in a quick analysis.
+
+## Proof of concept and pilot
+
+RMarkdown has been the tool of choice for authoring reports in the Courts project. DataMade staff familiar with Pweave have picked it up quickly and journalists without a deep background in programming have also been able to use it successfully (within the RStudio environment).
```

Comment:
It'd be great if we could link out to the relevant project here.

Reply:
these are not going to be accessible to all staff, let alone public folks. unfortunately.

Reply:
Pilots are useful in evaluating the tool, as well as for providing an example for future use. If we can't link to the project, could we host a clone [of the cookiecutter](https://gitlab.com/court-transparency-project/cookiecutter-court-transparency) as a basis for future analysis? It'd be ideal to add that in this repository, under `docker/templates/r-markdown` or something like that.